migrate.js

```diff
@@ -163,9 +166,15 @@ function cli () {
       type: 'boolean',
       default: defaults.quiet
     })
+    .option('disable-confs', {
+      alias: 'd',
+      describe: 'disable confirmation messages',
+      type: 'boolean',
+      default: defaults.disableconfs
+    })
     .option('force', {
       alias: 'f',
-      describe: 'disable confirmation messages',
+      describe: 'force deploy everything',
```

Comment:
It seems cleaner to me that there be a single option (promptDefault), rather than two (force and disable-confs).  The option "promptDefault" would take "yes" or "no".  It would hide all confirmations and supply the given response to all of them.

Reply:
That doesn't make much sense... A no answer for all means the script will do nothing.
Force just means true for all as you requested, and disable confirmations uses the default values.

Reply:
A no answer for all would mean the script will skip any contracts for which an address already exists.

A yes answer for all would mean the script will automatically rewrite any contracts for which an address already exists.